src/components/Header.js

```diff
@@ -0,0 +1,119 @@
+import React, {Component} from 'react';
+import {withStyles} from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
```

Comment:
I think it should be better if you seperate intenal and external imports.
Also i think that you can import @material-ui/core file and all components you need.
And in this case you'll avoid imports duplication.